shell/platform/fuchsia/flutter/integration_flutter_tests/embedder/child-view/child_view.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+import 'dart:async';
+
+TestApp app;
+
```

Comment:
small nit: this doesn't need to be async.

Reply:
done